javaslang/src-gen/main/java/javaslang/API.java

```diff
@@ -84,6 +90,865 @@ private API() {
+     * @param <T>      Component type of elements.
+     * @param elements Zero of more elements.
+     * @return A new Vector instance containing the given elements
+     * @throws NullPointerException if {@code elements} is null
+     */
+    @SuppressWarnings("varargs")
+    @SafeVarargs
+    public static <T> Vector<T> vector(T... elements) {
+      return Vector.of(elements);
+    }
+
+    /**
+     * Alias for {@link List#of(Object)}
+     *
+     * @param <T>     Component type of element.
+     * @param element An element.
+     * @return A new List instance containing the given element
+     */
+    public static <T> List<T> list(T element) {
+      return List.of(element);
+    }
+
+    /**
+     * Alias for {@link List#of(Object...)}
+     *
+     * @param <T>      Component type of elements.
+     * @param elements Zero of more elements.
+     * @return A new List instance containing the given elements
+     * @throws NullPointerException if {@code elements} is null
+     */
+    @SuppressWarnings("varargs")
+    @SafeVarargs
+    public static <T> List<T> list(T... elements) {
+      return List.of(elements);
+    }
+
+    /**
+     * Alias for {@link Stream#of(Object)}
+     *
+     * @param <T>     Component type of element.
+     * @param element An element.
+     * @return A new Stream instance containing the given element
+     */
+    public static <T> Stream<T> stream(T element) {
+      return Stream.of(element);
+    }
+
+    /**
+     * Alias for {@link Stream#of(Object...)}
+     *
+     * @param <T>      Component type of elements.
+     * @param elements Zero of more elements.
+     * @return A new Stream instance containing the given elements
+     * @throws NullPointerException if {@code elements} is null
+     */
+    @SuppressWarnings("varargs")
+    @SafeVarargs
+    public static <T> Stream<T> stream(T... elements) {
+      return Stream.of(elements);
+    }
+
+    /**
+     * Alias for {@link Queue#of(Object)}
+     *
+     * @param <T>     Component type of element.
+     * @param element An element.
+     * @return A new Queue instance containing the given element
+     */
+    public static <T> Queue<T> queue(T element) {
+      return Queue.of(element);
+    }
+
+    /**
+     * Alias for {@link Queue#of(Object...)}
+     *
+     * @param <T>      Component type of elements.
+     * @param elements Zero of more elements.
+     * @return A new Queue instance containing the given elements
+     * @throws NullPointerException if {@code elements} is null
+     */
+    @SuppressWarnings("varargs")
+    @SafeVarargs
+    public static <T> Queue<T> queue(T... elements) {
+      return Queue.of(elements);
+    }
+
+    /**
+     * Alias for {@link LinkedHashSet#of(Object)}
+     *
+     * @param <T>     Component type of element.
+     * @param element An element.
+     * @return A new LinkedHashSet instance containing the given element
+     */
+    public static <T> LinkedHashSet<T> linkedHashSet(T element) {
```

Comment:
@danieldietrich is the word `hash` part important here?
The user should care about the properties of the set (i.e. sorted, keeps insertion order etc), not whether we're using hashing or not